addons/account/wizard/account_move_send.py

```diff
@@ -227,6 +236,17 @@ def _get_mail_template_attachments_data(self, mail_template):
     # COMPUTE METHODS
     # -------------------------------------------------------------------------
 
+    @api.depends('move_ids')
+    def _compute_show_pdf_template_menu(self):
+        available_templates = self.env['ir.actions.report'].search([('is_invoice_report', '=', True)], limit=2)
+        for wizard in self:
+            if len(available_templates) == 1:
+                wizard.pdf_template_id = available_templates[0]
```

Comment:
Modifying a field in the compute of another is a bad smell, i would put the compute also for the pdf_template_id, and if you do so here you can use a search_count